src/styles/index.scss

```diff
@@ -1,3 +1,5 @@
+@import './blocks/product-card';
+
 body {
   margin: 0;
 }
```

Comment:
According to the checklist, there's a missing semicolon at the end of the `@import` statement. Make sure to end all statements with a semicolon for consistency and to avoid potential issues during compilation.